android/expoview/src/main/java/host/exp/exponent/notifications/NotificationHelper.java

```diff
@@ -372,9 +372,9 @@ public static void showNotification(
           final ExponentManifest exponentManifest,
           final Listener listener) {
     final ExponentNotificationManager manager = new ExponentNotificationManager(context);
-    final String experienceId = (String) details.get("experienceId");
+    final String experienceScopeKey = (String) details.get("experienceId");
```

Comment:
Looking at other call sites, I believe it's correct to use `NotificationConstants` to key into the details HashMap:

```suggestion
    final String experienceScopeKey = (String) details.get(NotificationConstants. NOTIFICATION_EXPERIENCE_ID_KEY);
```